specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -332,6 +405,7 @@ For MSIX, the COM activator GUID and the exe path need to be registered in the m
 ```
 
 # API Details
+
 ```c# (but really MIDL3)
```

Comment:
This is not valid syntax, let's not jam up the machines parsing this file.

```suggestion
```c#
```